src/psyclone/f2pygen.py

```diff
@@ -906,10 +906,11 @@ class BaseDeclGen(BaseGen):
                           DIMENSION(xx))
     :param bool allocatable: whether this declaration is for an \
                              ALLOCATABLE quantity
-    :param bool save: whether this declaration has the SAVE attribute
-    :param bool target: whether this declaration has the TARGET attribute
+    :param bool save: whether this declaration has the SAVE attribute.
+    :param bool target: whether this declaration has the TARGET attribute.
     :param initial_values: Initial value to give each variable.
     :type initial_values: list of str with same no. of elements as entity_decls
+    :param bool private: whether this declaration has the PRIVATE attribute.
```

Comment:
Could you add 'defaults to False'. I realise the other descriptions don't have this (but they should).

Reply:
Done.